clos.md

```diff
@@ -2,1655 +2,1556 @@
 title: Fundamentals of CLOS
 ---
 
-[Ravenbrook](http://www.ravenbrook.com/) / [ILC 2003](http://www.international-lisp-conference.org/)
-
-[Nick Levine](mailto:ndl@ravenbrook.com), [Ravenbrook Limited](http://www.ravenbrook.com/), 2003-07-15
 
+CLOS is the "Common Lisp Object System", arguably one of the most
+powerful object systems available in any language.
 
-## 1. Introduction
+Some of its features include:
 
-This document was written for presentation during a tutorial session at the
-[International Lisp Conference](http://www.international-lisp-conference.org/)
-held in New York City in October 2003.
+* it is **dynamic**, making it a joy to work with in a Lisp REPL. For
+  example, changing a class definition will update the existing
+  objects, given certain rules which we have control upon.
+* it supports **multiple dispatch** and **multiple inheritance**,
+* it is different from most object systems in that class and method
+  definitions are not tied together,
+* it has excellent **introspection** capabilities,
+* it is provided by a **meta-object protocol**, which provides a
+  standard interface to the CLOS, and allows *in fine* to create new
+  object systems.
 
-The intended audience for the tutorial is anybody with a basic
-knowledge of lisp or scheme, who wants to know something about how to
-use the "Common Lisp Object System" (CLOS). However, in an attempt to
-provide something for everyone, the tutorial will cover:
+The functionality belonging to this name was added to the Common Lisp
+language between the publication of Steele's first edition of "Common
+Lisp, the Language" in 1984 and the formalization of the language as
+an ANSI standard ten years later.
 
-* an introduction to the 10% of CLOS which you need to get you
-through 90% of use cases;
-* a number of opinionated statements, for instance about the
-differences between CLOS and other object systems;
-* a brief look "under the hood" at how some of the more dynamic
-features of CLOS might be implemented;
-* some exercises.
+This page aims to give a good understanding of how to use CLOS, but little presents of the MOP.
```

Comment:
Suggest "but little presents of the MOP" to "but only a brief introduction to the MOP"